src/easier68k/core/models/list_file.py

```diff
@@ -138,14 +158,93 @@ def load_from_json(self, json_str: str):
         loaded = json.loads(json_str)
         self.symbols = loaded['symbols']
         self.data = loaded['data']
+        self.starting_execution_address = loaded['startingExecutionAddress']
+
+    def read_s_record_filename(self, filepath: str):
+        """
+        Read the S record at the given file path
+        :param filepath: Path to an S record
+        :return:
+        """
+        with open(filepath, 'r') as f:
+            for line in f:
+                # process the line in the file
+                self.__process_s_record_line(line)
+
+    def __process_s_record_line(self, line: str):
+        """
+        Process a single line of the S record
+        This is defined here: http://www.easy68k.com/easy68ksrecord.htm
+        :param line:

```

Comment:
Incomplete docstring

Reply:
(I know, I'm being nitpicky, this is pretty self-explanatory: still though, at least having the return line filled out (even if it just says it doesn't return anything) would be helpful)

Reply:
I think having a blank return line indicates there's no return. I guess it does no harm either way though.